CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added socket option variant that enables the timestamp socket
   control message: `nix::sys::socket::sockopt::ReceiveTimestamp`
   ([#663](https://github.com/nix-rust/nix/pull/663))
+- Add nix::sys::fallocate
+  ([#596](https:://github.com/nix-rust/nix/pull/596))
```

Comment:
This should be `768` and not `596`. You can either guess on the PR number or just modify the PR after you submit adding this in the future.